src/AffectedUserFlow/CodeInput/CodeInputForm.tsx

```diff
@@ -144,7 +148,7 @@ const CodeInputForm: FunctionComponent = () => {
     }
   }
 
-  const isDisabled = code.length !== codeLength
+  const isDisabled = code.length !== codeLength || !code.match(/^\d+$/)
```

Comment:
You could also use `codeContainsNonDigitChars` here.

Reply:
It may also be nice to name `code.length !== codeLength`

```
const codeIsInvalidLength = code.length !== codeLength
const isDisabled = codeIsInvalidLength || codeContainsNonDigitChars
```